transition/doc/doc.md

```diff
@@ -1,14 +1,14 @@
 ```js script
-import '@divriots/dockit-core/sass-showcases';
-import { html } from '~/md-layout';
+import '@divriots/dockit-core/css-showcases';
 ```
 
 # Transition
 
 ## Time
 
-```html:html
-<dockit-sass-showcases
+```html story
+<dockit-css-showcases
+  mode="time"
```

Comment:
I had to add `mode="time"` here, because `--transition` has been deprecated in favor of `--time`
just pay attention to this and maybe consider actually changing the prefix to `--time`, but I didn't feel like such renaming belongs to this PR